### DISCUSSION TITLE
How can I set the reported output filename?

### DISCUSSION THREAD

**Discussion by chekrd** [biomejs/biome#3200] [Q&A] [answered]
AFAIK Biome prints everything to the console. Is there a way to let Biome create a reporter output file? Haven't found any output file config in the docs.
I am trying to use Junit reporter if it matters.

**Comment by ematipico** [comment, marked as answer]
There's not such option at the moment. For now, you'll have to it manually. We will consider this feature for the near future

**Comment by Conaclos** [comment]
You could use a shell redirection? I am not sure to see the interest of supporting another option for writing to a file...

**Reply by chekrd** [reply]
I have tried but it requires "a lot": Creating a directory if it does not exist and handling unexpected characters printed by Biome to the console. 

But never mind, JUnit reporter seems to be broken at this point. Will consider creating a bug for that.
